services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/QrCodePosterTemplateStructureProvider.java

```diff
@@ -70,7 +68,7 @@ private QRCodePosterTemplateAndroid buildAndroidProtoStructure(
         .setTemplate(qrTemplateLoader.loadAndroidTemplateAsBytes())
         .setQrCodeSideLength(templateConfig.getQrCodeSideLength())
         .setDescriptionTextBox(QRCodeTextBoxAndroid.newBuilder()
-            .setOffsetX(textBoxConfig.getOffsetX().floatValue()).setOffsetY(textBoxConfig.getOffsetY().floatValue())
+            .setOffsetX(textBoxConfig.getOffsetX().intValue()).setOffsetY(textBoxConfig.getOffsetY().intValue())
```

Comment:
That value is definitely a float (see [here](https://github.com/corona-warn-app/cwa-server/blob/release/2.0/common/protocols/src/main/proto/app/coronawarn/server/common/protocols/internal/pt/qr_code_poster_template.proto#L18)). Maybe something broke during merge?